### PR TITLE
Fix up DECQ Soyuz TM/TMA/MS Capsule

### PR DESCRIPTION
**Pull request by vader111**
Fix up DECQ Soyuz TM/TMA/MS Capsule.

Fixes Up https://github.com/KSP-RO/ROCapsules/pull/23

### CODE REVIEW

**Comment by vader111** [comment]
(Ready To Review)

File: GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/SwitchRotary_Solar.cfg [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+{
+  name = switchRotary_Solars
+  
+// Knob model
+ 
+  	MODEL
+	{
+		model = ASET/ASET_Props/Control/SwitchRotary/model/Switch_Rotary_Model
+		texture = SwitchRotaryDiffuse,ASET/ASET_Props/Control/SwitchRotary/model/SwitchRotaryDiffuse
+	}
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/SwitchRotary_Solar.cfg:11]: Does this entire PROP declaration require a :NEEDS[ASET] ?

File: GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/SwitchRotary_SoyIntL.cfg [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+ 
+	MODULE
+	{
+		name = JSIActionGroupSwitch
+		switchTransform = Switch_Rotary
+		switchSound = ASET/ASET_Props/Sounds/click-027
+		actionName = intlight
+		perPodPersistenceName = SWITCH_ROTARY_CLEAR_06
+		internalLightName = LIGHT
+	}
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/SwitchRotary_SoyIntL.cfg:37]: What other mods does this config require?  (ASET, JSI?)

File: GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/VZOR/prop.cfg [hunk 3]
```diff
@@ -0,0 +1,98 @@
+PROP
+{
+  name = VZOR
+	MODULE
+  	{
+	name = RasterPropMonitor
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/VZOR/prop.cfg:6]: What mods does this require?  Also please fix formatting

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSHeatshield.cfg [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	minimum_drag = 0.2
+	angularDrag = 2
+	crashTolerance = 16
+	breakingForce = 250
+	breakingTorque = 250
+	stagingIcon = DECOUPLER_HOR
+	stageOffset = 1
+	childStageOffset = 1
+
+	heatShieldDiameter	= 2.17
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSHeatshield.cfg:62]: Bad tab

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSOrbitalModule.cfg [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+//	==================================================
+//	TAC Life Support compatibility.
+//	==================================================
+
+@PART[ROC-SoyuzMSOM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@description ^= :$: Provides support resources for a crew of 3 for up to 30 days.
+
+	@mass -= 0.315
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSOrbitalModule.cfg:117]: Don't subtract out the mass of the resources from the part.  Just let it be empty space if needed.

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteMain.cfg [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+		}
+	}
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = RealChuteModule
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteMain.cfg:110]: :NEEDS[RealChute] ?

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteSpare.cfg [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+		}
+	}
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = RealChuteModule
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteSpare.cfg:111]: :NEEDS[RealChute]

**Review by DRVeyl — COMMENTED** [review]
Initial partial look.  Definitely needs testers!

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteSpare.cfg (hunk 7 above)
Comment (vader111) [reply GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteSpare.cfg:111]: Well considering Realchute is a requirement for RO is it required?

File: GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteMain.cfg (hunk 6 above)
Comment (vader111) [reply GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteMain.cfg:110]: Well considering Realchute is a requirement for RO is it required?

Comment (DRVeyl) [reply GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteMain.cfg:110]: Well, ROC is marked as not requiring RO.  Maybe a futile effort, that.

File: GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/Neptune/prop.cfg [hunk 8]
```diff
@@ -1,53 +1,57 @@
...
+	name = NEPTUNE4020
+
+	MODULE
+	{
+		name = RasterPropMonitor
+
+		screenTransform = INPU1_LCD
+
+		// Name of transform for the screen surface.
+		fontTransform = BobCatind/HOME2/Props/Neptune/NEPTUNE_font
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/Neptune/prop.cfg:12]: I don't think these are valid as expected paths?  (Bobcat Industries is a separate mod, right?)

**Comment by dgeastman** [comment]
Just tried testing this, forked master, pulled this into my fork, game seems fine, but once I try to open up the action menu for the DECQ Soyuz, it fails to open and action menus will no longer open for any part.

**Pull request closed without merging**